lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -636,6 +642,7 @@ def add_payment_method_or_subscription(xml, money, payment_method_or_reference,
         else
           add_address(xml, payment_method_or_reference, options[:billing_address], options)
           add_address(xml, payment_method_or_reference, options[:shipping_address], options, true)
+          optionally_add_line_item_data(xml, options)
```

Comment:
Correct, I think the previous method was an artifact of an earlier approach. Changed.